flybywiresim/aircraft
# fix(efb): display local charts when selected through pinned charts menu

Fixes #8710

## Summary of Changes
Changes the pinned charts logic to use `chartName` as an alternative value for `chartLinks` when no links exist on the `pinnedChart` object due to local files not using `chartLinks`.

## Screenshots (if necessary)
N/A

## References
N/A




## Additional context
Previously, the chart would not be displayed and instead the "chart" alt text would be shown until the user clicked the chart in the "Local Files" tab.


Discord username (if different from GitHub):

## Testing instructions
1. Add a PDF file to the `flybywire-externaltools-simbridge\resources\pdfs` folder
2. Load into MSFS and pin this file through the "Local Files" menu
3. Open the chart through the "Pinned Charts" menu
4. Chart should open as expected


## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page
